package/batocera/gpu/mali-g31-gbm/mali-g31-gbm.mk

```diff
@@ -24,4 +24,31 @@ endef
 MALI_G31_GBM_POST_INSTALL_STAGING_HOOKS += MALI_G31_GBM_COPY_KHRPLATFORM_STAGING
 endif
 
+ifeq ($(BR2_PACKAGE_BATOCERA_TARGET_ODROIDGOA),y)
```

Comment:
Perhaps we should check for `ifeq ($(BR2_cortex_a35),y)` instead?

Reply:
I'm not sure how odroid blobs would/could work on non-hardkernel rk3326 hardware, but it makes sense.